os/arch/arm/src/amebasmart/amebasmart_serial.c

```diff
@@ -1201,6 +1231,37 @@ static uint32_t rtk_uart_suspend(uint32_t expected_idle_time, void *param)
 {
 	(void)expected_idle_time;
 	(void)param;
+
+	ALIGNMTO(CACHE_LINE_SIZE) u8 flag[CACHE_LINE_ALIGMENT(64)];
+	ALIGNMTO(CACHE_LINE_SIZE) u32 uart_data[16];
+	IPC_MSG_STRUCT ipc_req_msg  __attribute__((aligned(64)));
+	ipc_req_msg.msg_type = IPC_USER_POINT;
+	ipc_req_msg.msg = (u32)uart_data;
+	ipc_req_msg.msg_len = sizeof(uart_data);
+	ipc_req_msg.rsvd = (u32)flag;
+
+	memset(flag, 0, sizeof(flag));
+	memset(uart_data, 0, sizeof(uart_data));
```

Comment:
1. Could you check performance difference between 0 initialization at variable define and memset like this?
2. What is the `flag` usage in this function?
3. You add little bit heavy operation in suspend and resume. How does they affect sleep and wakeup time? Could you check them for before and after? 
    1. when it enters sleep, before it takes xx ms, after yyms
    2. when it wakes up, before and after?